src/core_ghidra.cpp

```diff
@@ -271,7 +281,9 @@ static void Decompile(RCore *core, DecompileMode mode)
 		{
 			PJ *pj = pj_new ();
 			if(!pj)
+			{
```

Comment:
Please keep the code style as is, see https://gist.github.com/thestr4ng3r/d0b1de10852ca3d119e8c83375211e4b

Reply:
@thestr4ng3r Please review again